README.md

```diff
@@ -9,26 +9,25 @@ Read more [here](http://datafordemocracy.org/projects/ethics.html).
 ## Ethics Principles
 ---
 <ul>
-    <li>Its my job to understand, mitigate and communicate the presence of bias in algorithms.</li>
-    <li>Be responsible for maximizing social benefit and minimizing harm.</li>
-    <li>Practice humility and openness.</li>
+    <li>Bias will exist. It's my responsibility to understand, mitigate and communicate the presence of bias in algorithms.</li>
+    <li>It's my responsibility to increase social benefit while minimizing harm.</li>
+    <li>I will practice humility and openness.</li>
     <li>I will know my data and help future users know it as well.</li>
-    <li>Make reasonable efforts to know and document its origins and document its transformation.</li>
-    <li>Bias will exist. Measure it. Plan for it.</li>
-    <li>Thou shalt document transparently, accessibly, responsibly,  reproducibly, and communicate.</li>
-    <li>Engaging the whole community. Do you have all relevant individuals engaged?</li>
-    <li>People before data - data scientists should use a question driven approach rather than a data-driving or methods approach. Consider personal safety and treat others the way they want to be treated.</li>
-    <li>Exercise ethical imagination.</li>
-    <li>Open by default  - use of data should be transparent and fair.</li>
+    <li>I will make reasonable efforts to know and document its origins and document its transformation.</li>
+    <li>I will document transparently, accessibly, responsibly,  reproducibly, and communicate.</li>
+    <li>I will strive to engage "the whole community" using, or impacted by, my work. I will try to clarify or redefine the definition of this community, on an ongoing basis. I will use skepticism and an open mind to expand this definition.</li>
+    <li>I will put people before data.</li>
     <li>I will not over/under represent findings.</li>
-    <li>You are part of an ecosystem  understand context and provenance.</li>
-    <li>Respecting human dignity.</li>
-    <li>Respect their data even more than your own. Understand where its sources and think about the consequences of your actions.</li>
-    <li>Protecting individual and institutional privacy.</li>
+    <li>Data scientists should use a _question-driven approach_ rather than a _data-driven_ or _methods_ approach. Data scientists should consider personal safety and <a href="https://en.wikipedia.org/wiki/Golden_Rule">treat others the way they want to be treated.</a></li>
```

Comment:
I tried to add a reference for "Treat others the way they want to be treated," as it's kind of an unstable notion - one should not project that others have the same respect or disrespect for privacy. Consider that some actors at privacy-disrespecting companies do so because they (_think_) that they do not care about their own privacy (the old, "what do I have to hide?" attitude).

I would opt for removing this "treat others the way..." principle completely   - substituting something with a more rigorous definition - but in any case it could be referenced or defined.

Reply:
There was much discussion about this point. Personally, I 'm not in favor of removing the  principle that covers human dignity. Human dignity and respect of the individual and their human rights should be the umbrella under which all things we make are shaded, if you will. I do not think that the addition, while a solid addition```Data scientists should use a _question-driven approach_ rather than a _data-driven_ or _methods_ approach. Data scientists should consider personal safety and treat others the way they want to be treated.``` covers this adequately nor is it similar enough to replace the ref to Human dignity.

Reply:
I'd love to hear others viewpoints on this tho, as the above is just my personal opinion, obviously :)

Reply:
@cheapwebmonkey The respect-human-dignity one is just moved actually, see here: https://github.com/Data4Democracy/ethics-resources/pull/14/files#r180514002